app/models/dialog_field.rb

```diff
@@ -13,6 +13,7 @@ class DialogField < ApplicationRecord
                                   :message => "Field Name %{value} is reserved."}
 
   default_value_for :required, false
+  default_value_for :visible, true
```

Comment:
We either need a migration to initialize this field to `true` or change default_value_for to:
  `default_value_for(:visible, :allows_nil => false) { true }`

@Fryguy @jrafanie Thoughts on this one?  I noticed we do not use `:allows_nil => false` anywhere else.